packages/web/webpack/index.html

```diff
@@ -4,9 +4,23 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="icon" type="image/png" href="logo.png" />
```

Comment:
Nit but I don't think you need the closing / for link

Reply:
I'm seeing some mixed info about whether that's still the case for html5 (vs html4)